src/containers/Tenant/utils/schemaActions.ts

```diff
@@ -34,23 +35,44 @@ const upsertQueryTemplate = (path: string) => {
 VALUES ( );`;
 };
 
+interface ActionsOptions {
+    enableAdditionalQueryModes?: boolean;
+    setQueryMode?: (mode: QueryMode) => void;
+}
+
 const bindActions = (
     path: string,
     dispatch: Dispatch<any>,
     setActivePath: (path: string) => void,
+    options?: ActionsOptions,
```

Comment:
I'd better rename this prop to `additionalEffects` maybe...

Reply:
I don't like `additionalEffects` name. I'll call it just `additionalParams`